frontend/awx/resources/inventories/hooks/useInventoriesGroupsActions.tsx

```diff
@@ -0,0 +1,35 @@
+import { useTranslation } from 'react-i18next';
+import { AwxGroup } from '../../../interfaces/InventoryGroup';
+import {
+  IPageAction,
+  PageActionSelection,
+  PageActionType,
+  usePageNavigate,
+} from '../../../../../framework';
+import { useMemo } from 'react';
+import { PencilAltIcon } from '@patternfly/react-icons';
+import { AwxRoute } from '../../../main/AwxRoutes';
+import { cannotEditResource } from '../../../../common/utils/RBAChelpers';
+
+export function useInventoriesGroupsActions() {
+  const { t } = useTranslation();
+  const pageNavigate = usePageNavigate();
+
+  return useMemo<IPageAction<AwxGroup>[]>(
+    () => [
```

Comment:
Consider adding the `Run Command` button to the row items.  Perhaps reach out to UXD to see what they think.  The button doesn't have to be functional in this PR as there is an issue for it here https://issues.redhat.com/browse/AAP-19957